.github/workflows/ghcr.yml

```
@@ -16,12 +19,17 @@ jobs:
 
     steps:
     # Checkout do código
+    - name: Harden Runner
```

yamllint

[warning] wrong indentation: expected 6 but found 4

(indentation)